tests/test_output_formatter.cpp

```diff
@@ -318,6 +318,33 @@ auto construct_find_multiple_reply_duplicate_image()
 
     return reply;
 }
+
+auto construct_version_info_multipassd_update_available()
+{
+    auto reply = mp::VersionReply();
+    reply.set_version("Daemon version");
+    reply.set_log_line("Some logline goes here");
```

Comment:
Why this?

Reply:
The `log_line` is redundant, you are correct, but the `version` is used to drive testable output `multipassd: ` in the test cases.

Reply:
Yeah I meant just the log line.